packages/ExternalLink/stories/examples/LayoutExamples.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Story } from "storybook/assets/styles/common.styles";
+import ExternalLink from "../../src";
+
+const ExampleStory = () => (
+  <Story>
+    <p>
+      This is default <code>ExternalLink</code> style:{" "}
+      <ExternalLink ariaText="aria text for link" href="http://acl-ui.aclgrc-s1.com" />
```

Comment:
Want to update these links to something more contemporary? 😄

Reply:
Maybe. :) Ok Ok

Reply:
Don't waste a good rickroll opportunity!